assets/stylesheets/sass/components/_fonts.scss

```diff
@@ -0,0 +1,48 @@
+@font-face {
+    font-family: 'Merriweather';
+    src: url('../../../fonts/merriweather-bold-webfont.eot');
+    src: url('../../../fonts/merriweather-bold-webfont.eot?#iefix') format('embedded-opentype'),
+         url('../../../fonts/merriweather-bold-webfont.woff2') format('woff2'),
+         url('../../../fonts/merriweather-bold-webfont.woff') format('woff'),
+         url('../../../fonts/merriweather-bold-webfont.ttf') format('truetype'),
+         url('../../../fonts/merriweather-bold-webfont.svg#merriweatherbold') format('svg');
+    font-weight: bold;
+    font-style: normal;
+}
+
+
+@font-face {
+    font-family: 'Merriweather';
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings